shared/middleware/helpers/assets.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Assets
+#
+class AssetBucket
+
+  ALLOWED_FILE_TYPES = %w(jpg jpeg gif png mp3)
```

Comment:
This looks like the same listed of allowed filed types as in assets_api.rb.  Is it only coincidentally the same, or should assets_api.rb use this constant rather than defining its own?